resources/views/common/h2-ie-dialog.html.twig

```diff
@@ -0,0 +1,72 @@
+<div data-h2-system>
+  <div
+    aria-hidden="true"
+    aria-describedby="ieDialogContent"
+    aria-labelledby="ieDialogTitle"
+    data-h2-dialog="ieDialog"
+    tabindex="-1"
+    role="dialog">
+    <div
+      data-h2-dialog-wrapper
+      data-h2-radius="b(round)">
+      <div
+        data-h2-dialog-title
+        data-h2-bg-color="b(theme-2, 1)"
+        data-h2-padding="b(tb, 1) b(left, 1) b(right, 3)">
+        <h5
+          data-h2-focus
+          data-h2-font-color="b(white)"
+          id="ieDialogTitle">Hey! We noticed you're using Internet Explorer.</h5>
+        <button
+          data-h2-dialog-exit-trigger
+          data-h2-button="white, round, medium, outline"
+          type="button">
+          <span>x</span>
+        </button>
+      </div>
+      <div
+        data-h2-dialog-content
+        data-h2-padding="b(all, 1)">
+        <div id="ieDialogContent">
+          <p>Thanks for visiting Talent Cloud!</p>
+          <p data-h2-margin="b(tb, 1)">As <a href="" title="">Microsoft has begun phasing out support for Internet Explorer</a>, we recommend using Talent Cloud from a modern web browser for the best experience. We recommend looking at any of the following:</p>
```

Comment:
@bitliege Should the `href` and `title` attributes in the `a` tag not have values or is this forthcoming?